installer/install.iss

```diff
@@ -34,7 +34,11 @@ LZMAUseSeparateProcess=yes
 OutputBaseFilename={#FILENAME_VERSION}
 OutputDir={#GetEnv('TEMP')}
 #else
+#if IS_ARM64==1
+OutputBaseFilename={#APP_NAME+'-'+FILENAME_VERSION}-arm64
+#else
 OutputBaseFilename={#APP_NAME+'-'+FILENAME_VERSION}-{#BITNESS}-bit
+#endif
```

Comment:
Hmm. It would probably make more sense to make this more generic: it is simply a file name suffix. Something like this:

```c
#ifdef INSTALLER_FILENAME_SUFFIX
 OutputBaseFilename={#APP_NAME+'-'+FILENAME_VERSION-INSTALLER_FILENAME_SUFFIX}
#else
OutputBaseFilename={#APP_NAME+'-'+FILENAME_VERSION}-{#BITNESS}-bit
#endif
```

And then `#define INSTALLER_FILENAME_SUFFIX 'arm64'` in `config.iss` instead.